### ISSUE TITLE
JSFiddle does not work on Safari anymore

### ISSUE THREAD

**Issue by victornpb** [jsfiddle/jsfiddle-issues#1133]
None of the panels appear. None of the buttons on both the sidebar and top menu work.

![screen shot 2018-01-10 at 11 18 25 pm](https://user-images.githubusercontent.com/3372598/34803887-a04fd286-f66d-11e7-8552-e30292a41d9a.png)

Errors on console:
```
[Error] SyntaxError: Unexpected identifier 'currentInstance'
	(anonymous function) (_dist-editor.js:1)
[Error] ReferenceError: Can't find variable: Cookie
	global code (jsfiddle.net:271)
[Error] TypeError: window.addEvent is not a function. (In 'window.addEvent', 'window.addEvent' is undefined)
	global code (jsfiddle.net:1692)
[Error] TypeError: info.addEvent is not a function. (In 'info.addEvent("click", function(){
                      info.className = "";
                    })', 'info.addEvent' is undefined)
	(anonymous function) (jsfiddle.net:814)
```

Safari Version 9.1.2 (11601.7.7)

**Comment by oskarkrawczyk** [comment]
@victornpb Hard to say why you're getting this.

Can you try hard refresh cache for jsfiddle.net?

**Comment by oskarkrawczyk** [comment]
Ah, ok it's Safari 9 (current version is 11), which is really old and doesn't support any ES6.

I'm planning on fixing some stuff for IE11 which should probably fix this old Safari issues.

Relevant #1137

**Comment by victornpb** [comment]
As the date of this comment, I can confirm that **it is working** now (Safari Version 9.1.2 (11601.7.7)). I haven't used it for extended period of time, but at least it is visually right.

Safari 9 is the latest version available for OS X El Capitan 10.11.6 (15G31) I would update the browser, but since it requires an OS update, I'm not ready to do it yet.
